server/fleet/apple_mdm.go

```diff
@@ -151,22 +151,26 @@ type MDMAppleDEPKeyPair struct {
 	PrivateKey []byte `json:"private_key"`
 }
 
-// MDMAppleCommandResult holds the result of a command execution provided by the target device.
+// MDMAppleCommandResult holds the result of a command execution provided by
+// the target device.
 type MDMAppleCommandResult struct {
-	// ID is the enrollment ID. This should be the same as the device ID.
-	ID string `json:"id" db:"id"`
+	// DeviceID is the enrollment ID. This should be the same as the device ID.
+	DeviceID string `json:"device_id" db:"device_id"`
```

Comment:
this is the host uuid, right?

Reply:
Yes, should I name it `HostUUID` in the struct? It's `DeviceID` because it is read from the nano enrollment tables where it is named as such, but for our use case anyway it should/must always be the host UUID.

Reply:
I don't have a strong opinion honestly, I was mostly confused by the comment (which already existed)

Reply:
Ok I'll at least clarify the comment.